### ISSUE TITLE
SELECT failed with additional partitioning on integer column

### ISSUE THREAD

**Issue by the20login** [timescale/timescaledb#66]
SELECT failed if condition contains filter over integer column, wich was used as additional partitioning column.

```sql
CREATE TABLE history (
  timestamp TIMESTAMPTZ,
  object_id int,
  value int
);
SELECT create_hypertable('history', 'timestamp', 'object_id', 10);
SELECT * FROM history WHERE object_id = 1;
```
resulting:
```sql
ERROR:  function _timescaledb_catalog.get_partition_for_key(integer, integer) does not exist
```

Version 0.0.11-beta from dockerhub

**Comment by cevian** [comment]
This bug should be fixed once pr #69 is merged.

**Comment by the20login** [comment]
OK, thanks.
